### PR TITLE
README Formatting

### PR DESCRIPTION
**Pull request by ElectronicsArchiver**
[Badge Suggestions]: https://img.shields.io/badge/Ｒｅｐｏｓｉｔｏｒｙ_Ｓｕｇｇｅｓｔｉｏｎｓ-4b781e?style=for-the-badge&logoColor=white&logo=GitHub&labelColor=609926
[Badge Changes]: https://img.shields.io/badge/Ｃｈａｎｇｅｓ-9e122a?style=for-the-badge&logoColor=white&logo=GitLFS&labelColor=D81939
[Badge GitHub]: https://img.shields.io/badge/ＧｉｔＨｕｂ_Ｎｅｗｃｏｍｅｒｓ-be2e47?style=for-the-badge&logoColor=white&logo=BitWarden&labelColor=FD3A5C
[Badge Robot]: https://img.shields.io/badge/Ｉ＇ｍ_Ｈｕｍａｎ-8c3b63?style=for-the-badge&logoColor=white&logo=RuboCop&labelColor=c84f8c
[Badge Ask]: https://img.shields.io/badge/Ｐｌｅａｓｅ_Ａｓｋ_Ｍｅ-4e2760?style=for-the-badge&logoColor=white&logo=AskUbuntu&labelColor=73398D
[Badge Info]: https://img.shields.io/badge/Ｉｎｆｏｒｍａｔｉｏｎ-01434e?style=for-the-badge&logoColor=white&logo=InternetArchive&labelColor=006272

[Button MarkedDown]: https://img.shields.io/badge/MarkedDown-00b2ff?style=for-the-badge&logoColor=white&logo=Markdown

[License Badges]: https://github.com/MarkedDown/Licenses
[Widgets]: https://github.com/MarkedDown/Disable/blob/main/Features/Widgets.md
[Tabs]: https://github.com/MarkedDown/Disable/blob/main/Features/Tabs.md
[Wiki]: https://github.com/MarkedDown/Wiki
[MarkedDown]: https://github.com/MarkedDown 'How to better use Markdown'

<br>
<br>

<div align = center>

[<kbd> <br> Please Check The **Preview** <br> </kbd>][Preview]

</div>
  

<br>
<br>

## ![Badge Changes]

- General formatting / spacing

- Added quicklinks

<br>
<br>

## ![Badge Ask]

- ***If you have any questions.***

- ***In case you do not agree with some of the changes made in this PR***,
  *please first let me know about them, before disregarding it entirely so*
  *we can possibly find a middle ground and have it not go to waste.*

<br>
<br>

## ![Badge Info]

If you want to know more about **Markdown** check out:

<br>

                            [![Button MarkedDown]][MarkedDown]

<br>
<br>

[Preview]: https://github.com/ElectronicsArchiver/opengloves-driver/tree/R2

### CODE REVIEW

**Comment by ElectronicsArchiver** [comment]
@danwillm Could you please review this PR~

**Comment by danwillm** [comment]
Sorry, have been away for a while due to examinations otherwise would have merged this in.

We're currently working on a refactor to the codebase to include a lot more features so this might be outdated soon if we merge it now. Do you mind if we leave it open for another week or so until we've got the refactor done, then we can talk more about what might be useful to add to the readme? Thanks for this though, it looks good :)

**Comment by ElectronicsArchiver** [comment]
Well you could also merge it and in a week I could adjust it?
Sounds like only the content, such as features, will be modified?

File: README.md [hunk 1]
```diff
@@ -1,46 +1,90 @@
...
 
-- **Multiple Communication Methods** <br>
-    *- Bluetooth Serial* <br>
-    *- Named Pipes* <br>
-    *- Serial USB*
+-   **Button / Joystick Inputs**
 
-### Additional Tools
+    - <kbd>      Trigger      </kbd>  <kbd>  A  </kbd>  <kbd>  B  </kbd> <br>
+    - <kbd>  Joystick  </kbd>  <kbd>  X  </kbd>  <kbd>  Y  </kbd>  <kbd>  Click  </kbd>
```
Comment (danwillm) [inline README.md]: Not a big fan of what these look like... I think how they were previously was fine.

File: README.md [hunk 2]
```diff
@@ -1,46 +1,90 @@
...
+    
+    <br>
+    
+- **Force Feedback**
+
+    <br>
```
Comment (danwillm) [inline README.md]: can we reduce the spacing between these?

File: README.md [hunk 3]
```diff
@@ -50,42 +94,52 @@ To make your own hardware compatible with **OpenGloves**, refer to the **[Driver
 ***[Issue]*** *or contact us first to discuss* <br>
 *what you would like to change.*
 
----
+<br>
+<br>
 
-## Credits
+## Maintainer
```
Comment (danwillm) [inline README.md]:
```suggestion
## Maintainers
```

**Review by danwillm — CHANGES_REQUESTED** [review]
just a few comments

File: README.md (hunk 1 above)
Comment (ElectronicsArchiver) [reply README.md]: That's unfortunate, kbd is even made for that~

**Comment by ElectronicsArchiver** [comment]
??

**Pull request closed without merging**